data_taking_scripts/motor_control.py

```diff
@@ -54,17 +54,18 @@ def inch_to_cm(dist):
         #log narrowscan
         logger.log_modemap(narrow_scan_start_freq, narrow_scan_stop_freq, sec_wait_for_na_averaging, 'narrowscan')
 
-        mirror_spacing_tracker, new_plate_separation = orpheus_motors.move_by_increment(increment_distance,
-                                                                                   plate_thickness,
-                                                                                   mirror_spacing_tracker,
-                                                                                   num_plates,
-                                                                                   initial_plate_separation)
-        orpheus_motors.wait_for_motors()
         i = round((i+inch_to_cm(increment_distance)),4)
-        current_resonator_length = current_resonator_length+inch_to_cm(increment_distance)
-        initial_plate_separation = new_plate_separation
+        if i <= abs(distance_to_move):
```

Comment:
@parashar2399 why do you need this statement? Doesn't the while statement on line 46 make this line redundant?

Reply:
you are right. I just need to move the increment of the counter to the start of the while loop. otherwise by the time it realizes that its overshot the mark, it has already moved the motors.